service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseControllerTest.java

```diff
@@ -187,163 +235,125 @@ void shouldThrowAnExceptionIfCaseDoesNotContainGeneratedOrders() {
 
         @Test
-        void shouldRemoveThirdGeneratedOrderAndNotModifyChildren() {
-            Element<Child> childElement = element(childrenId, Child.builder()
-                .party(ChildParty.builder()
-                    .firstName("Tom")
-                    .lastName("Wilson")
-                    .build())
-                .finalOrderIssuedType("Yes")
-                .finalOrderIssued("Yes")
-                .build());
+        void shouldRemoveC21FromFirstHearingOrderBundle() {
+            HearingOrder cmoOrder = HearingOrder.builder()
+                .type(HearingOrderType.DRAFT_CMO)
+                .status(CMOStatus.DRAFT)
+                .order(testDocumentReference())
+                .build();
 
-            List<Element<Child>> children = newArrayList(childElement);
+            HearingOrder orderToBeRemoved = HearingOrder.builder()
+                .type(HearingOrderType.C21)
+                .status(CMOStatus.SEND_TO_JUDGE)
+                .order(testDocumentReference())
+                .build();
 
-            Element<GeneratedOrder> orderOne = element(orderOneId, generateOrder(BLANK_ORDER));
-            Element<GeneratedOrder> orderTwo = element(orderTwoId, generateOrder(BLANK_ORDER));
-            Element<GeneratedOrder> orderThree = element(orderToBeRemovedId, generateOrder(CARE_ORDER, children));
+            Element<HearingOrdersBundle> firstHearingBundle = element(UUID.randomUUID(), HearingOrdersBundle.builder()
+                .orders(Lists.newArrayList(element(ID_ONE, cmoOrder),
+                    element(ID_TWO, orderToBeRemoved))).build());
 
-            List<Element<GeneratedOrder>> orderCollection = newArrayList(orderOne, orderTwo, orderThree);
+            Element<HearingOrdersBundle> secondHearingBundle = element(UUID.randomUUID(), HearingOrdersBundle.builder()
+                .orders(Lists.newArrayList(element(ID_ONE, cmoOrder),
+                    element(ID_TWO, orderToBeRemoved))).build());
+
+            List<Element<HearingOrdersBundle>> hearingOrdersBundlesDrafts = List.of(firstHearingBundle, secondHearingBundle);
+
+            CaseDetails caseDetails = caseDetailsWithHearingOrdersBundle(migrationId, familyManNumber, hearingOrdersBundlesDrafts);
 
-            CaseDetails caseDetails = caseDetails(migrationId, familyManNumber, orderCollection, children);
             CaseData extractedCaseData = extractCaseData(postAboutToSubmitEvent(caseDetails));
 
-            assertThat(extractedCaseData.getOrderCollection()).isEqualTo(List.of(orderOne, orderTwo));
-            assertThat(extractedCaseData.getChildren1()).isEqualTo(children);
-            assertThat(extractedCaseData.getHiddenOrders()).isEmpty();
+            assertThat(extractedCaseData.getHearingOrdersBundlesDrafts().get(0).getValue().getOrders()).isEqualTo(Lists.newArrayList(element(ID_ONE, cmoOrder)));
+            assertThat(extractedCaseData.getHearingOrdersBundlesDrafts().get(1)).isEqualTo(secondHearingBundle);
```

Comment:
I'd prefer an assert on the entire object e.g `assertThat(extractedCaseData.getHearingOrdersBundlesDrafts()).isEqualTo(List.of(...));` but if let's keep since it's urgent.